vsm

```diff
@@ -547,6 +843,8 @@ if __name__ == "__main__":
                 logger.e(err)
                 exit(1)
 
+        config_tree = TreeNode(NODE_ROOT, None)
```

Comment:
It serves as the document root so we have a parent for the arbitrary number of top-level list items.